lib/iex/lib/iex/helpers.ex

```diff
@@ -325,9 +325,26 @@ defmodule IEx.Helpers do
 
   @doc """
   Retrieves the nth expression's value from the history.
+  Where nth is iex(nth)>
 
-  Use negative values to look up expression values relative to the current one.
-  For instance, v(-1) returns the result of the last evaluated expression.
+  It is usefull to bound expression on the history to a variable
+
+  ## Examples
+
+      iex(1)> 40 + 2
+      42
+      iex(2)> [1, 2, third | rest] = [1, 2, 3, 4, 6, 7, 8, 9]
+      [1, 2, 3, 4, 6, 7, 8, 9]
+      iex(3)> third
+      3
+      iex(4)> rest
+      [4, 6, 7, 8, 9]
+      iex(5)> v(2)
+      [1, 2, 3, 4, 6, 7, 8, 9]
+      iex(6)> v(1)
+      42
+      iex(7)> meanning_of_life = v
+      42
```

Comment:
Thanks for the quick response :) 
I mean assign when i say bound, i am learning elixir and i hear every where that equals is a patern match operator not really assign sorry for a misunderstood for bound,
but i think that need better example. because this https://youtu.be/5kYmOyJjGDM?t=2h15m32s